src/libraries/Common/tests/System/Collections/DebugView.Tests.cs

```diff
@@ -139,6 +139,14 @@ private static IEnumerable<object[]> TestDebuggerAttributes_ListInputs()
             yield return new object[] { new Dictionary<float, double> { { 1.0f, 1.0 }, { 2.0f, 2.0 } }.Values };
             yield return new object[] { new SortedDictionary<Guid, string> { { Guid.NewGuid(), "One" }, { Guid.NewGuid(), "Two" } }.Keys };
             yield return new object[] { new SortedDictionary<long, Guid> { { 1L, Guid.NewGuid() }, { 2L, Guid.NewGuid() } }.Values };
+#if !NETFRAMEWORK
```

Comment:
I find it surprising that we're running collections tests against .NET framework here. @stephentoub do you know if this is intentional or could we just remove it?

Reply:
This file is used by the immutable collection tests which run on netfx because immutable collections have a specialized netstandard/netfx build.